odoo/orm/models.py

```diff
@@ -6774,35 +6774,32 @@ def _transient_vacuum(self):
         - the 10 rows that have been created/changed the last 5 minutes will NOT
           be deleted
         """
+        remaining = 0
         if self._transient_max_hours:
             # Age-based expiration
-            self._transient_clean_rows_older_than(self._transient_max_hours * 60 * 60)
+            remaining = self._transient_clean_rows_older_than(self._transient_max_hours * 60 * 60)
```

Comment:
Shouldn't it be `remaining +=`? so we accumulate the remaining of the two underlying transient methods, and `_transient_clean_old_rows` doesn't come and erase the result of `_transient_clean_rows_older_than`. I know it is a bit silly because nobody (sane) is using `--osv-memory-count-limit` but still...